ksicht/core/templates/core/solution_submit.html

```diff
@@ -13,12 +13,16 @@ <h2 class="heading is-size-7">{{ task.title }} úloha</h2>
             {% if form %}
                 {% crispy form %}
             {% else %}
-                <article class="message is-warning is-size-6">
-                    <div class="message-body">
-                        <p>Pro tuto úlohu jste již <strong>odevzdal/a</strong> řešení <strong>{{ submission.file|filename }}</strong>.</p>
-                        <p>KSICHT umožňuje odeslat řešení <strong>pouze jednou</strong>.</p>
-                    </div>
-                </article>
+            <article class="message is-warning is-size-6">
+                <div class="message-body">
```

Comment:
Ještě si říkám, že v případě, že už nejde řešení smazat (`accepts_solution_submissions` nebo tak nějak), bylo by vhodné o tom uživatele nějak informovat. Rozhodně asi není úplně vhodný mu tam poskytnout link na smazání, který pak ale nezafunguje.

Reply:
Pokud to chápu správně tak na solution solution_submit.html se uživatel dostane jen, když může zároveň i mazat (accepts_solution_submissions = True).  Nemůže mazat, když už je po deadline. ale to by se neměl dostat ani na solution_submit.html, nebo když řešení není jeho (chybné id, ale to musí zadat sám do url).

Reply:
Nojo, asi máš pravdu. OK, zavírám.